CHANGELOG.md

```diff
@@ -57,6 +57,8 @@ Emojis for the following are chosen based on [gitmoji](https://gitmoji.dev/).
 - Code was refactored to work with the new enum style of command state management.
 - Enums are now used to control switching between conjugations.
 - Enums are now used to control switching between different conjugation displays.
+- Light and dark mode colors are now defined in `Assets.xcassets` and accessed via `ScribeColor.getter:color` or `UIColor`'s new convenience initializer.
+- Variants of the Scribe key icon are placed into `Assets.xcassets`, making it unnecessary to check for light/dark mode and device type in code.
```

Comment:
> I updated the Changelog as well after merging in the current master branch into my branch. However, I am not sure if I placed it under the correct version. I wasn't sure if 2.0.0 is already released now.

Hey @misteu! Wanted to add a quick note regarding the above :arrow_up: so that what you brought up doesn't get overlooked :+1:. Looks like [2.0.0](https://github.com/scribe-org/Scribe-iOS/releases/tag/2.0.0) did already get released, only couple days ago. However, it does look like @andrewtavis did just recently add now the next version in the `CHANGELOG.md` as well over [here](https://github.com/scribe-org/Scribe-iOS/blob/main/CHANGELOG.md#scribe-ios-210). My guess is that what you added would go in there instead.

@andrewtavis - just a thought that I had, but could it be good to note which version is currently in development? e.g. the title of the current subsection in the `CHANGELOG.md` could perhaps be 

### [Unreleased] Scribe-iOS 2.1.0 

or

### [In Progress] Scribe-iOS 2.1.0

or even just simply

### [Unreleased]

or something else of the like. Just an idea, since I think it could help clarify, in case other contributors wonder the same thing. Also, since there can often be cleanup of the `CHANGELOG.md` right before a release is made regardless too, I figured it could be easy enough to do the rename to the actual version title then. 

Have a great weekend all!

Reply:
I totally agree, @wkyoshida :) I think that simply marking it as `[Unreleased]` if we're unsure if it's going to be a minor or a patch release, and as soon as we figure out what kind it is we can add in the version. The upcoming one adds at least one feature, so we can mark it as `[Unreleased] Scribe-iOS 2.1.0` for now 😊

Thanks for your further contributions, and hope both of you are having a nice weekend so far!

Reply:
[521976d](https://github.com/scribe-org/Scribe-iOS/pull/228/commits/521976dd54db99cf62ee80686365292a200c50a8) makes the changes that we discussed here 😊